code/datums/world_topic.dm

```diff
@@ -174,6 +174,14 @@
 
 	return jointext(message, "")
 
+/datum/world_topic/voice_announce
+	keyword = "voice_announce"
```

Comment:
Should probably require comms key to prevent somebody from playing or loading into memory any file on the host system.

Reply:
Comms key is unnecessary, when you click the URL it generates a unique ID, and as well as that it generates a token to go with that and the webserver has acess too that... yes, I was too lazy to put the comms key in the webserver config

Reply:
Is the user not able to extract the unique ID generated and the call the topic endpoint with that ID & any file they want?

Reply:
The unique ID is in the URL, but the token is never sent to the user.

Reply:
Oh I see, the token isnt shared with the user but is instead stored in a file that both the webadmin and byond have access to... a bit of a roundabout method, but it works I guess.